src/pages/Swap/useGetSwapInfo/index.ts

```diff
@@ -0,0 +1,82 @@
+import BigNumber from 'bignumber.js';
+import { useMemo } from 'react';
+import { Token } from 'types';
+import { convertTokensToWei } from 'utilities';
+
+import { Swap } from '../types';
+
+// TODO: move to global constants
```

Comment:
We can resolve this TODO in this PR

Reply:
I see there are other todos in this PR that relate to this so it could also wait